style.css

```diff
@@ -0,0 +1,21 @@
+img {
+  width: 500px;
+}
+a {
+  text-decoration: none;
+}
+a:hover {
+  color: rgb(116, 101, 101);
+}
+.noodles-recipe {
```

Comment:
Try to organise CSS styles in a coherent manner so that the file doesn't get messy :)